lib/linguist.rb

```diff
@@ -15,9 +15,9 @@ class << Linguist
   #       see Linguist::LazyBlob and Linguist::FileBlob for examples
   #
   # Returns Language or nil.
-  def detect(blob)
+  def detect(blob, options={})
```

Comment:
I was about to say "we don't do that anywhere else and maybe there's a compat reason", but I just found a single instance, so yes!